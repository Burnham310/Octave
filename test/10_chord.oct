main = Section ;; {
  1  [1 3 5]  6'/8  5'/16
  [3 5]  4  [4 6]'/8  3'/8
  1  [1 3 5]  6'/8  5'/16
  [2 4]  3'/8  2'/8  1'/16
  
  [5 7]  [4 6]'/8  5'/16  [3 5]'/8
  1  [1 3 5]  6'/8  5'/16
  1'/8  [1 4 6]  4'/16  [2 5]'/8
  5  4  [3 5]'/8  2'/16

}
