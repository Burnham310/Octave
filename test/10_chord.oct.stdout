[
{
  "freq": 130.80999755859375,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.5,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 164.81027221679688,
  "duration": 0.5,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 195.99354553222656,
  "duration": 0.5,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 0,
  "inst": "guitar"
}
,
{
  "freq": 219.9953155517578,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 195.99354553222656,
  "duration": 0.125,
  "gap": 0.125,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 164.81027221679688,
  "duration": 0.5,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 195.99354553222656,
  "duration": 0.5,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 0,
  "inst": "guitar"
}
,
{
  "freq": 174.6103973388672,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 174.6103973388672,
  "duration": 0.25,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 219.9953155517578,
  "duration": 0.25,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0,
  "inst": "guitar"
}
,
{
  "freq": 164.81027221679688,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.5,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 164.81027221679688,
  "duration": 0.5,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 195.99354553222656,
  "duration": 0.5,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 0,
  "inst": "guitar"
}
,
{
  "freq": 219.9953155517578,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 195.99354553222656,
  "duration": 0.125,
  "gap": 0.125,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 146.82925415039062,
  "duration": 0.5,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 174.6103973388672,
  "duration": 0.5,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 0,
  "inst": "guitar"
}
,
{
  "freq": 164.81027221679688,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 146.82925415039062,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.125,
  "gap": 0.125,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 195.99354553222656,
  "duration": 0.5,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 246.93638610839844,
  "duration": 0.5,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 0,
  "inst": "guitar"
}
,
{
  "freq": 174.6103973388672,
  "duration": 0.25,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 219.9953155517578,
  "duration": 0.25,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0,
  "inst": "guitar"
}
,
{
  "freq": 195.99354553222656,
  "duration": 0.125,
  "gap": 0.125,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 164.81027221679688,
  "duration": 0.25,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 195.99354553222656,
  "duration": 0.25,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0,
  "inst": "guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.5,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 164.81027221679688,
  "duration": 0.5,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 195.99354553222656,
  "duration": 0.5,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 0,
  "inst": "guitar"
}
,
{
  "freq": 219.9953155517578,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 195.99354553222656,
  "duration": 0.125,
  "gap": 0.125,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.5,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 174.6103973388672,
  "duration": 0.5,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 219.9953155517578,
  "duration": 0.5,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 0,
  "inst": "guitar"
}
,
{
  "freq": 174.6103973388672,
  "duration": 0.125,
  "gap": 0.125,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 146.82925415039062,
  "duration": 0.25,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 195.99354553222656,
  "duration": 0.25,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0,
  "inst": "guitar"
}
,
{
  "freq": 195.99354553222656,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 174.6103973388672,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 164.81027221679688,
  "duration": 0.25,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 195.99354553222656,
  "duration": 0.25,
  "gap": 0,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 130.80999755859375,
  "duration": 0.25,
  "gap": 0.25,
  "amp": 0,
  "inst": "guitar"
}
,
{
  "freq": 146.82925415039062,
  "duration": 0.125,
  "gap": 0.125,
  "amp": 1,
  "inst": "guitar"
}
,
{
  "freq": 0,
  "duration": 0,
  "gap": 0,
  "amp": 0,
  "inst": "guitar"
}

]